lndgrpc/aio/async_client.py

```diff
@@ -132,6 +132,15 @@ async def send_payment(self, payment_request):
             response = await self._ln_stub.SendPaymentSync(request)
             return response
 
+        # ROUTERRPC
+        @handle_rpc_errors
+        async def send_payment_v2(self, payment_request):
+            """Send a payment over lightning"""
+            request = router.SendPaymentRequest(payment_request=payment_request,**kwargs)
```

Comment:
There are no **kwargs in the `send_payment_v2` method to pass through here - I think it needs to be removed.

Reply:
I should probably remove this async work. 

Anyways though, I use **kwargs a lot throught the pr, it allows the user to specify arbitrary inputs to the function. That way we dont have to update our code/ bloat out code with arguments for every possible input variable names, we can just define required ones, but allows any others to be passed in an used.